### DISCUSSION TITLE
PhD scholarship in Digital Representative Microstructure Generation with Generative Artificial Intelligence

### DISCUSSION THREAD

**Discussion by frankNiessen** [ORTools4MTEX/ORTools#68] [Announcements]
Dear **ORTools** community, 

I would like to advertise an upcoming **PhD** opportunity. The fully-funded 3 year PhD project is part of the research project **Generative AI in Crystal Orientation Space** which is led by myself and involves close collaboration with **Ralf Hielscher** from **MTEX**. You will work on the forefront of implementing **generative AI** in crystal orientation space to generate super-realistic **digital representative microstructures**, enabling entirely **digital materials design** workflows. With this project you will directly contribute to the development of advanced digital materials design to realize fast and superior materials solutions, addressing global societal needs of the future.

The PhD project interfaces computational materials science with generative artificial intelligence and will be supervised at **DTU Construct** and **DTU Compute**. You will furthermore have the unique opportunity to work with esteemed international collaboration partners at **TU Bergakademie Freiberg** (Ralf Hielscher) and **UC Santa Barbara** (Tresa Pollock, McLean Echlin).

I am looking for curious and motivated candidates with a solid background in **materials**, **physics**, **computer science** and/or **generative artificial intelligence**. The application deadline is **16th of February**. Find additional information and details on the application procedure right [here](https://efzu.fa.em2.oraclecloud.com/hcmUI/CandidateExperience/en/sites/CX_1/job/3011/).

**Comment by Globle-Thunder** [comment]
Hi, 

![IMG_0049](https://github.com/ORTools4MTEX/ORTools/assets/98560737/e8d2e79c-69b3-4e69-999e-48d091e20765)
![IMG_0050](https://github.com/ORTools4MTEX/ORTools/assets/98560737/d98c70c4-c823-49fd-96e6-4bee150080b1)
the result to be generated should be a three-dimensional structural diagram of the crystal, or it can be similar to the image below, including textual information about the crystal orientation space.

**Reply by frankNiessen** [reply]
Hi Globle-Thunder, 

I can unfortunately not see the images you attached. The generated data should be an orientation map, or, eventually, an orientation 3D volume. In essence the aim is to generate representative microstructures such as currently enabled with Neper or Dream3D (see [right here](https://neper.info/_images/gene_gene.png)), but hopefully more realistic. 

Best wishes
Frank

**Reply by Globle-Thunder** [reply]
Hi,

Thanks.  The images are in the attachment. Yes, I got it.

First, use generative model to create a file containing the microstructure of crystals, similar to the .cif file shown in the image.

Then, use third-party software (Neper or Dream3D) or plugins (pymatgen) to generate diagrams of the crystal structures.
![cif](https://github.com/ORTools4MTEX/ORTools/assets/98560737/8b7feb25-e3ac-4f97-ac54-0873962f38af)
![ZnSi2Sb](https://github.com/ORTools4MTEX/ORTools/assets/98560737/36ab5904-21bf-4b0a-a9f9-73aebd9fa4a4)

Best regards,
Liu
